react/wbooks/src/app/components/SignUp/layout.js

```diff
@@ -8,13 +8,30 @@ import woloxLogoImage from '../../assets/wolox-logo.png';
 import InputLabel from '../InputLabelStateless';
 import ErrorMessages from '../ErrorMessages';
 import { Routes } from '../../../constants';
+import { LANGUAGES } from '../../config/i18n';
 
 import styles from './styles.module.scss';
-import { LOGIN, SIGN_UP, FIELDS } from './constants';
+import { LOGIN, SIGN_UP, FIELDS, EN, ES } from './constants';
 
-function SignUp({ handleSubmit, errorMessages, isError }) {
+function SignUp({ handleSubmit, errorMessages, isError, handleLanguageChangeClick }) {
   return (
     <div className={`${styles.container} column background-wild-sand`}>
+      <div className="row">
+        <button
+          type="button"
+          className={styles.esButton}
+          onClick={() => handleLanguageChangeClick(LANGUAGES.ES)}
```

Comment:
use something like `data-lang`, and assign the click to a handler like this
```
const handleChangeLanguage = (event) => { handleLanguageClick(event.target.dataset.lang) }
```
so you don't repeat and use arrow function inside an onClick assign :)